mythril/analysis/modules/state_change_external_calls.py

```diff
@@ -0,0 +1,125 @@
+from mythril.analysis.ops import Variable, VarType
+from mythril.analysis.swc_data import REENTRANCY
+from mythril.analysis.modules.base import DetectionModule
+from mythril.analysis.report import Issue
+from mythril.analysis.call_helpers import get_call_from_state
+from mythril.analysis.modules.external_calls import CallIssue, ExternalCallsAnnotation
```

Comment:
I think different modules shouldn't depend on one another. Now it's still relatively simple, but later on this will become super hard to maintain

Reply:
Yes, I was thinking we can place all the annotations in 1 file(probably that might solve the future problems of this sort). WDYT?

Reply:
I'd prefer having them in each of the modules. 
That keeps them self contained, and understandable. 

I don't think the limited amount of code reuse is worth sacrificing that bit of understandability